### PR TITLE
Update access and add automated scanning

### PR DESCRIPTION
**Pull request by mira-miracoli**
details here:
 - https://github.com/usegalaxy-eu/issues/issues/514

I removed some people form the table, from which I think they are not in this scope anymore, please feel free to change this if my guess was wrong.

### CODE REVIEW

File: _data/pii.yml [hunk 1]
```diff
@@ -59,18 +56,20 @@ data:
       reason: Generated as a result of your use of the system
       handling: |
           We store the data securely in our database which only the
-          UseGalaxy.eu administrators have access to (BG, SJ, HR)
+          UseGalaxy.eu administrators have access to.
+
+          We perform automated security scans on job data to protect our systems against
```
Comment (bgruening) [inline _data/pii.yml:61]: Is it important that we only do that for a subset of tools?

Comment (mira-miracoli) [reply _data/pii.yml:61]: We can specify it, but I think that would only limit us and I would assume we would need to specify which tools exactly and maybe warn the users in the toolbox directly. (but just guessing)

**Review by JanLeendertse — APPROVED** [review]
Sounds good to me.

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**